### DISCUSSION TITLE
How to see logical page size of a PDF document

### DISCUSSION THREAD

**Discussion by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#2295] [Q&A] [answered]
From old Discourse Forum 

https://forum.sumatrapdfreader.org/t/how-to-tell-if-a-document-is-a4-or-letter-size/4698

It appears current pre-release and 3.3.3 no longer show nearest page size, this should be reporting A4

![image](https://user-images.githubusercontent.com/12754913/151633400-cc7ba998-6e42-47f1-ac5e-e4552689cecc.png)

in 3.2 and earlier it would show as A4

![image](https://user-images.githubusercontent.com/12754913/151633290-4569530f-6b8c-4570-8651-739f6d20c182.png)

There are times when its handy for a non familiar size to be shown as say Letter or Statement etc.

**Comment by kjk** [comment, marked as answer]
Looks like a regression, I've opened https://github.com/sumatrapdfreader/sumatrapdf/issues/2297

**Reply by kjk** [reply]
Should be fixed now.

**Reply by GitHubRulesOK** [reply]
That was quick I hope it was not related to paper print sizes?

**Reply by kjk** [reply]
No, just a logical typo switching `<` for `>` and as a result never finding logical paper size for given paper dimensions.

**Reply by GitHubRulesOK** [reply]
Great just literally downloaded and works just as expected.
